src/SFML/Graphics/Texture.cpp

```diff
@@ -441,6 +454,94 @@ void Texture::update(const Uint8* pixels, unsigned int width, unsigned int heigh
 
 
 ////////////////////////////////////////////////////////////
+void Texture::update(const Texture& texture)
+{
+    // Update the whole texture
+    update(texture, 0, 0);
+}
+
+
+////////////////////////////////////////////////////////////
+void Texture::update(const Texture& texture, unsigned int x, unsigned int y)
```

Comment:
Wouldn't it be better to just take the offset by accepting `sf::Vector2u` instead? You'll always have both params anyway. You could also just pass `copy.getSize()` in line 112.